testmodels/src/main/java/com/amplifyframework/testmodels/parenting/schema.graphql

```diff
@@ -0,0 +1,25 @@
+type Parent @model {
+  id: ID!
+  name: String!
+  address: Address!
+  children: [Child]
+}
+
+type Child {
+	name: String!
```

Comment:
Can you align the formatting in this file, so all blocks use all +2 or all +4? (What do the other .graphql files in the code base currently use?)

Reply:
+ 2